lang/interpret_test/TestAstFunc2/escaping4.output

```diff
@@ -0,0 +1 @@
+# err: err8: parser failed: cannot parse string: X: \z
```

Comment:
IMO `invalid escape sequence`

Reply:
true, sadly it's not guaranteed that we know why. TBH I'm trying to merge the ragel bits in a "good enough" state and hopefully a smart ragel person fixes the things I got wrong by adding fancy errors to it all.